docs/patterns/single-new-eks-observability-accelerators/single-new-eks-awsnative-fargate-observability.md

```diff
@@ -158,9 +158,168 @@ Example with "EKS_Cluster" metrics
 
 Although the default metrics exposed by cloudWatchAdotAddon are useful for getting some standardized metrics from our application we often instrument our own application with OLTP to expose metrics. Fortunately, the otel-collector-cloudwatch-collector can be specified as the endpoint for collecting these metrics and getting metrics and logs to cloudwatch. 
 
-We will be fetching metrics from `ho11y` a synthetic signal generator allowing you to test observability solutions for microservices. It emits logs, metrics, and traces in a configurable manner. The application can be deployed using `team-geordie` ArgoCD configuration found [here](https://github.com/aws-observability/aws-observability-accelerator/tree/main/artifacts/argocd-apps/teams/team-geordie).
-
-We will have to edit the `OTEL_EXPORTER_OTLP_ENDPOINT` value in `ho11y.yaml` and set the value as 'otel-collector-cloudwatch-collector.default.svc.cluster.local:4317' 
+We will be fetching metrics from `ho11y` a synthetic signal generator allowing you to test observability solutions for microservices. It emits logs, metrics, and traces in a configurable manner. 
+
+## Deploying Workload
```

Comment:
Close the loop properly here. Save this to a file and and then apply it like Java Workload example.